test/lib/rule-finder.js

```diff
@@ -2,13 +2,41 @@ const path = require('path');
 const createRequire = require('create-require');
 const assert = require('assert');
 const proxyquire = require('proxyquire');
-
+const { builtinRules } = require('eslint/use-at-your-own-risk');
 const semver = require('semver');
-const eslintPkg = require('eslint/package.json');
+const { ESLint } = require('eslint');
 
 const processCwd = process.cwd;
+const isV8Eslint = ESLint.configType === 'eslintrc';
+const eslintVersion = isV8Eslint ? 'prior-v8' : 'post-v8';
+
+const isObject = (item) => {
+  return item && typeof item === 'object' && !Array.isArray(item);
+};
+
+const deepMerge = (target, source) => {
```

Comment:
lodash.merge is fine if we need deep merging - I’m not sure why inlining it would be better. My previous comment was because it didn’t seem to me like we needed deep merging.

Reply:
Ok, so back to the implementation in lodash.merge.